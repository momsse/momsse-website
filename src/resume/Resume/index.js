import React, { PropTypes } from 'react'

import Job from '../Job'
import technologies from '../../core/enums/technologies'
import Project from '../Project'
import { Container } from './styledComponents'

const Resume = () => (
  <Container>
    <Job
      enterprise="Chauffeur privé"
      position="Senior Software engineer"
      startDate={new Date('Nov 01 2015')}
    >
      <Project
        title="Driver onboarding"
        abstract="initialization de la stack et developpement de la web app d'onboarding chauffeur"
        technologies={[
          technologies.Javascript,
          technologies.NodeJS,
          technologies.Express,
          technologies.ReactJS,
          technologies.Webpack,
        ]}
      />

      <Project
        title="Sauron (real time driver and booking manager)"
        abstract="initialization de la stack et developpement de la web app de gestion de la flotte chauffeurs."
        technologies={[
          technologies.Javascript,
          technologies.NodeJS,
          technologies.Express,
          technologies.ReactJS,
          technologies.Webpack,
          technologies.Bootstrap,
          technologies.SocketIO,
          technologies.IndexedDB,
          technologies.MongoDB,
        ]}
      />

      <Project
        title="Driver state (real time driver's state)"
        abstract="initialization de la stack et developpement de l'API d'état des chauffeurs"
        technologies={[
          technologies.Javascript,
          technologies.NodeJS,
          technologies.Express,
          technologies.RabbitMQ,
          technologies.MongoDB,
        ]}
      />

      <Project
        title="Business backend"
        abstract="migration de l'API monolithique vers une architecture micro services."
        technologies={[
          technologies.Javascript,
          technologies.NodeJS,
          technologies.MongoDB,
        ]}
      />
    </Job>

    <Job
      enterprise="Ekino"
      position="Software engineer"
      startDate={new Date('Jul 01 2014')}
      endDate={new Date('Oct 01 2015')}
    >
      <Project
        title="Coorpacademy"
        abstract="Application d'e-learning (MOOC)."
        technologies={[
          technologies.Javascript,
          technologies.NodeJS,
          { ...technologies.AngularJS, version: '1.x' },
          technologies.MongoDB,
        ]}
      />

      <Project
        title="SFR"
        abstract="Backend d'une application de gestion des ventes de forfaits mobile et fixes."
        technologies={[
          { ...technologies.Java, version: '8' },
          technologies.Spring,
          { ...technologies.Tomcat, version: '8' },
          technologies.Oracle,
        ]}
      />

      <Project
        title="SFR"
        abstract="Backend d'une application mobile assistant les vendeurs lors des ventes en boutique physique."
        technologies={[
          { ...technologies.Java, version: '8' },
          { ...technologies.Jersey, version: '2' },
          { ...technologies.Spring, version: '4' },
          { ...technologies.Tomcat, version: '8' },
          technologies.Oracle,
        ]}
      />

      <Project
        title="SFR"
        abstract="API permettant la collecte d'information des visiteurs lors de leurs passages en boutique physique et/ou en ligne."
        technologies={[
          { ...technologies.Java, version: '7' },
          { ...technologies.Spring, version: '4' },
          technologies.SpringBatch,
          { ...technologies.Tomcat, version: '7' },
          technologies.MongoDB,
        ]}
      />
    </Job>

    <Job
      enterprise="Groupe Onepoint"
      position="Software engineer"
      startDate={new Date('Nov 01 2011')}
      endDate={new Date('Jun 01 2014')}
    >
      <Project
        title="Ministère de la Justice"
        abstract=" TMA de l'ensemble applicatif de gestion de la chaîne pénale (KCOP)."
        technologies={[
          { ...technologies.Java, verison: '6' },
          technologies.Struts,
          { ...technologies.EJB, verison: '3' },
          technologies.Hibernate,
          technologies.JBoss,
          technologies.Oracle,
        ]}
      />

      <Project
        title="BMW"
        abstract="Développement de la 2nd version du portail du groupe."
        technologies={[
          { ...technologies.Java, version: '6' },
          technologies.JSF,
          { ...technologies.EJB, version: '3.1' },
          technologies.Glassfish,
          technologies.Oracle,
        ]}
      />

      <Project
        title="SFR"
        abstract="Refonte de l'application de facturation/comptabilité du groupe SFR (RefCCOP). "
        technologies={[
          { ...technologies.Java, version: '6' },
          technologies.Wicket,
          technologies.Spring,
          technologies.Hibernate,
          technologies.Tomcat,
          technologies.Oracle,
        ]}
      />
    </Job>

    <Job
      enterprise="Scalian"
      position="Student software engineer"
      startDate={new Date('Apr 01 2011')}
      endDate={new Date('Sep 01 2011')}
    >
      <Project
        title="Youkan"
        abstract="Développement de la troisième version d’une application web de gestion de projet agile en ligne. "
        technologies={[
          { ...technologies.Java, version: '6' },
          { ...technologies.Flex, version: '4.5' },
          technologies.Spring,
          technologies.iBatis,
          technologies.Tomcat,
          technologies.PostgreSQL,
        ]}
      />
    </Job>

    <Job
      enterprise="Université de Bordeaux"
      position="Master Génie Logiciel "
      startDate={new Date('Sep 01 2006')}
      endDate={new Date('Oct 01 2011')}
    >
      <Project
        abstract="Préparation d'une Licence puis d'un Master spécialité génie logiciel à l'université de Bordeaux (M. Bien)."
      />
    </Job>
  </Container>
)

export default Resume
