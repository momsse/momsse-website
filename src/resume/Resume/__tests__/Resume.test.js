import React from 'react'
import renderer from 'react-test-renderer'
import Resume from '../index'

describe('<Resume />', () => {
  it('should render successfully', () => {
    const tree = renderer.create(<Resume />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
