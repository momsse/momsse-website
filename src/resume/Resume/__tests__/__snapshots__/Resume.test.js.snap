// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Resume /> should render successfully 1`] = `
.c1 {
  padding: 0.5rem;
  background-color: #F0F0F0;
  border: 1px solid #d4d4d4;
  box-shadow: 0 1px 6px rgba(0,0,0,0.175);
}

.c2 {
  font-size: 1.25rem;
  margin-bottom: 0.25rem;
}

.c3 {
  color: #7f807a;
  font-size: 0.75rem;
  margin-bottom: 0.5rem;
}

.c4 {
  font-size: 0.75rem;
}

.c5 {
  font-weight: bold;
}

.c6::after {
  content: ' ';
}

.c7 {
  color: #86994d;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c7:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c0 > div {
  margin-bottom: 1rem;
}

@media print {
  .c7 {
    color: darken(#86994d,25%);
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      Senior Software engineer
       (
      Chauffeur privé
      )
    </div>
    <div
      className="c3"
    >
      <svg
        fill="currentColor"
        height="1em"
        preserveAspectRatio="xMidYMid meet"
        style={
          Object {
            "color": undefined,
            "verticalAlign": "middle",
          }
        }
        viewBox="0 0 40 40"
        width="1em"
      >
        <g>
          <path
            d="m4.4 37.1h6.4v-6.4h-6.4v6.4z m7.8 0h7.2v-6.4h-7.2v6.4z m-7.8-7.8h6.4v-7.2h-6.4v7.2z m7.8 0h7.2v-7.2h-7.2v7.2z m-7.8-8.6h6.4v-6.4h-6.4v6.4z m16.4 16.4h7.1v-6.4h-7.1v6.4z m-8.6-16.4h7.2v-6.4h-7.2v6.4z m17.2 16.4h6.4v-6.4h-6.4v6.4z m-8.6-7.8h7.1v-7.2h-7.1v7.2z m-7.9-19.3v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.4q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.4q0.3 0 0.5-0.2t0.2-0.5z m16.5 19.3h6.4v-7.2h-6.4v7.2z m-8.6-8.6h7.1v-6.4h-7.1v6.4z m8.6 0h6.4v-6.4h-6.4v6.4z m0.7-10.7v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.5q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.5q0.2 0 0.5-0.2t0.2-0.5z m8.5-1.4v28.5q0 1.2-0.8 2.1t-2 0.8h-31.4q-1.2 0-2.1-0.9t-0.8-2v-28.5q0-1.2 0.8-2t2.1-0.9h2.8v-2.1q0-1.5 1.1-2.6t2.5-1h1.4q1.5 0 2.5 1.1t1.1 2.5v2.1h8.6v-2.1q0-1.5 1-2.6t2.5-1h1.5q1.4 0 2.5 1.1t1 2.5v2.1h2.9q1.1 0 2 0.9t0.8 2z"
          />
        </g>
      </svg>
       
      novembre 2015
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          Driver onboarding
           :
        </span>
         
        initialization de la stack et developpement de la web app d'onboarding chauffeur
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://developer.mozilla.org/fr/docs/Web/JavaScript"
          target="_blank"
        >
          Javascript
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://nodejs.org/en/"
          target="_blank"
        >
          NodeJS
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://expressjs.com"
          target="_blank"
        >
          Express
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://reactjs.org/"
          target="_blank"
        >
          ReactJS
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://webpack.js.org/"
          target="_blank"
        >
          Webpack
          
        </a>
      </span>
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          Sauron (real time driver and booking manager)
           :
        </span>
         
        initialization de la stack et developpement de la web app de gestion de la flotte chauffeurs.
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://developer.mozilla.org/fr/docs/Web/JavaScript"
          target="_blank"
        >
          Javascript
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://nodejs.org/en/"
          target="_blank"
        >
          NodeJS
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://expressjs.com"
          target="_blank"
        >
          Express
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://reactjs.org/"
          target="_blank"
        >
          ReactJS
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://webpack.js.org/"
          target="_blank"
        >
          Webpack
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://getbootstrap.com/"
          target="_blank"
        >
          Bootstrap
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://socket.io/"
          target="_blank"
        >
          SocketIO
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://developer.mozilla.org/fr/docs/Web/API/API_IndexedDB"
          target="_blank"
        >
          IndexedDB
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.mongodb.com/"
          target="_blank"
        >
          MongoDB
          
        </a>
      </span>
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          Driver state (real time driver's state)
           :
        </span>
         
        initialization de la stack et developpement de l'API d'état des chauffeurs
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://developer.mozilla.org/fr/docs/Web/JavaScript"
          target="_blank"
        >
          Javascript
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://nodejs.org/en/"
          target="_blank"
        >
          NodeJS
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://expressjs.com"
          target="_blank"
        >
          Express
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.rabbitmq.com/"
          target="_blank"
        >
          RabbitMQ
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.mongodb.com/"
          target="_blank"
        >
          MongoDB
          
        </a>
      </span>
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          Business backend
           :
        </span>
         
        migration de l'API monolithique vers une architecture micro services.
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://developer.mozilla.org/fr/docs/Web/JavaScript"
          target="_blank"
        >
          Javascript
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://nodejs.org/en/"
          target="_blank"
        >
          NodeJS
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.mongodb.com/"
          target="_blank"
        >
          MongoDB
          
        </a>
      </span>
    </div>
  </div>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      Software engineer
       (
      Ekino
      )
    </div>
    <div
      className="c3"
    >
      <svg
        fill="currentColor"
        height="1em"
        preserveAspectRatio="xMidYMid meet"
        style={
          Object {
            "color": undefined,
            "verticalAlign": "middle",
          }
        }
        viewBox="0 0 40 40"
        width="1em"
      >
        <g>
          <path
            d="m4.4 37.1h6.4v-6.4h-6.4v6.4z m7.8 0h7.2v-6.4h-7.2v6.4z m-7.8-7.8h6.4v-7.2h-6.4v7.2z m7.8 0h7.2v-7.2h-7.2v7.2z m-7.8-8.6h6.4v-6.4h-6.4v6.4z m16.4 16.4h7.1v-6.4h-7.1v6.4z m-8.6-16.4h7.2v-6.4h-7.2v6.4z m17.2 16.4h6.4v-6.4h-6.4v6.4z m-8.6-7.8h7.1v-7.2h-7.1v7.2z m-7.9-19.3v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.4q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.4q0.3 0 0.5-0.2t0.2-0.5z m16.5 19.3h6.4v-7.2h-6.4v7.2z m-8.6-8.6h7.1v-6.4h-7.1v6.4z m8.6 0h6.4v-6.4h-6.4v6.4z m0.7-10.7v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.5q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.5q0.2 0 0.5-0.2t0.2-0.5z m8.5-1.4v28.5q0 1.2-0.8 2.1t-2 0.8h-31.4q-1.2 0-2.1-0.9t-0.8-2v-28.5q0-1.2 0.8-2t2.1-0.9h2.8v-2.1q0-1.5 1.1-2.6t2.5-1h1.4q1.5 0 2.5 1.1t1.1 2.5v2.1h8.6v-2.1q0-1.5 1-2.6t2.5-1h1.5q1.4 0 2.5 1.1t1 2.5v2.1h2.9q1.1 0 2 0.9t0.8 2z"
          />
        </g>
      </svg>
       
      juillet 2014 - octobre 2015
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          Coorpacademy
           :
        </span>
         
        Application d'e-learning (MOOC).
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://developer.mozilla.org/fr/docs/Web/JavaScript"
          target="_blank"
        >
          Javascript
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://nodejs.org/en/"
          target="_blank"
        >
          NodeJS
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://angular.io/"
          target="_blank"
        >
          AngularJS
           1.x
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.mongodb.com/"
          target="_blank"
        >
          MongoDB
          
        </a>
      </span>
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          SFR
           :
        </span>
         
        Backend d'une application de gestion des ventes de forfaits mobile et fixes.
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/java/"
          target="_blank"
        >
          Java
           8
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://spring.io/"
          target="_blank"
        >
          Spring
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://tomcat.apache.org/"
          target="_blank"
        >
          Tomcat
           8
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/database/index.html"
          target="_blank"
        >
          Oracle
          
        </a>
      </span>
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          SFR
           :
        </span>
         
        Backend d'une application mobile assistant les vendeurs lors des ventes en boutique physique.
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/java/"
          target="_blank"
        >
          Java
           8
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://jersey.github.io/"
          target="_blank"
        >
          Jersey
           2
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://spring.io/"
          target="_blank"
        >
          Spring
           4
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://tomcat.apache.org/"
          target="_blank"
        >
          Tomcat
           8
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/database/index.html"
          target="_blank"
        >
          Oracle
          
        </a>
      </span>
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          SFR
           :
        </span>
         
        API permettant la collecte d'information des visiteurs lors de leurs passages en boutique physique et/ou en ligne.
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/java/"
          target="_blank"
        >
          Java
           7
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://spring.io/"
          target="_blank"
        >
          Spring
           4
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://spring.io/projects/spring-batch"
          target="_blank"
        >
          SpringBatch
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://tomcat.apache.org/"
          target="_blank"
        >
          Tomcat
           7
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.mongodb.com/"
          target="_blank"
        >
          MongoDB
          
        </a>
      </span>
    </div>
  </div>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      Software engineer
       (
      Groupe Onepoint
      )
    </div>
    <div
      className="c3"
    >
      <svg
        fill="currentColor"
        height="1em"
        preserveAspectRatio="xMidYMid meet"
        style={
          Object {
            "color": undefined,
            "verticalAlign": "middle",
          }
        }
        viewBox="0 0 40 40"
        width="1em"
      >
        <g>
          <path
            d="m4.4 37.1h6.4v-6.4h-6.4v6.4z m7.8 0h7.2v-6.4h-7.2v6.4z m-7.8-7.8h6.4v-7.2h-6.4v7.2z m7.8 0h7.2v-7.2h-7.2v7.2z m-7.8-8.6h6.4v-6.4h-6.4v6.4z m16.4 16.4h7.1v-6.4h-7.1v6.4z m-8.6-16.4h7.2v-6.4h-7.2v6.4z m17.2 16.4h6.4v-6.4h-6.4v6.4z m-8.6-7.8h7.1v-7.2h-7.1v7.2z m-7.9-19.3v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.4q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.4q0.3 0 0.5-0.2t0.2-0.5z m16.5 19.3h6.4v-7.2h-6.4v7.2z m-8.6-8.6h7.1v-6.4h-7.1v6.4z m8.6 0h6.4v-6.4h-6.4v6.4z m0.7-10.7v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.5q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.5q0.2 0 0.5-0.2t0.2-0.5z m8.5-1.4v28.5q0 1.2-0.8 2.1t-2 0.8h-31.4q-1.2 0-2.1-0.9t-0.8-2v-28.5q0-1.2 0.8-2t2.1-0.9h2.8v-2.1q0-1.5 1.1-2.6t2.5-1h1.4q1.5 0 2.5 1.1t1.1 2.5v2.1h8.6v-2.1q0-1.5 1-2.6t2.5-1h1.5q1.4 0 2.5 1.1t1 2.5v2.1h2.9q1.1 0 2 0.9t0.8 2z"
          />
        </g>
      </svg>
       
      novembre 2011 - juin 2014
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          Ministère de la Justice
           :
        </span>
         
         TMA de l'ensemble applicatif de gestion de la chaîne pénale (KCOP).
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/java/"
          target="_blank"
        >
          Java
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://struts.apache.org/"
          target="_blank"
        >
          Struts
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://fr.wikipedia.org/wiki/Enterprise_JavaBeans"
          target="_blank"
        >
          EJB
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://hibernate.org/"
          target="_blank"
        >
          Hibernate
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://www.jboss.org/"
          target="_blank"
        >
          JBoss
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/database/index.html"
          target="_blank"
        >
          Oracle
          
        </a>
      </span>
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          BMW
           :
        </span>
         
        Développement de la 2nd version du portail du groupe.
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/java/"
          target="_blank"
        >
          Java
           6
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://www.oracle.com/technetwork/java/javaee/javaserverfaces-139869.html"
          target="_blank"
        >
          JSF
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://fr.wikipedia.org/wiki/Enterprise_JavaBeans"
          target="_blank"
        >
          EJB
           3.1
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://www.oracle.com/technetwork/middleware/glassfish/overview/index.html"
          target="_blank"
        >
          Glassfish
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/database/index.html"
          target="_blank"
        >
          Oracle
          
        </a>
      </span>
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          SFR
           :
        </span>
         
        Refonte de l'application de facturation/comptabilité du groupe SFR (RefCCOP). 
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/java/"
          target="_blank"
        >
          Java
           6
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://wicket.apache.org/"
          target="_blank"
        >
          Wicket
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://spring.io/"
          target="_blank"
        >
          Spring
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://hibernate.org/"
          target="_blank"
        >
          Hibernate
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://tomcat.apache.org/"
          target="_blank"
        >
          Tomcat
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/database/index.html"
          target="_blank"
        >
          Oracle
          
        </a>
      </span>
    </div>
  </div>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      Student software engineer
       (
      Scalian
      )
    </div>
    <div
      className="c3"
    >
      <svg
        fill="currentColor"
        height="1em"
        preserveAspectRatio="xMidYMid meet"
        style={
          Object {
            "color": undefined,
            "verticalAlign": "middle",
          }
        }
        viewBox="0 0 40 40"
        width="1em"
      >
        <g>
          <path
            d="m4.4 37.1h6.4v-6.4h-6.4v6.4z m7.8 0h7.2v-6.4h-7.2v6.4z m-7.8-7.8h6.4v-7.2h-6.4v7.2z m7.8 0h7.2v-7.2h-7.2v7.2z m-7.8-8.6h6.4v-6.4h-6.4v6.4z m16.4 16.4h7.1v-6.4h-7.1v6.4z m-8.6-16.4h7.2v-6.4h-7.2v6.4z m17.2 16.4h6.4v-6.4h-6.4v6.4z m-8.6-7.8h7.1v-7.2h-7.1v7.2z m-7.9-19.3v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.4q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.4q0.3 0 0.5-0.2t0.2-0.5z m16.5 19.3h6.4v-7.2h-6.4v7.2z m-8.6-8.6h7.1v-6.4h-7.1v6.4z m8.6 0h6.4v-6.4h-6.4v6.4z m0.7-10.7v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.5q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.5q0.2 0 0.5-0.2t0.2-0.5z m8.5-1.4v28.5q0 1.2-0.8 2.1t-2 0.8h-31.4q-1.2 0-2.1-0.9t-0.8-2v-28.5q0-1.2 0.8-2t2.1-0.9h2.8v-2.1q0-1.5 1.1-2.6t2.5-1h1.4q1.5 0 2.5 1.1t1.1 2.5v2.1h8.6v-2.1q0-1.5 1-2.6t2.5-1h1.5q1.4 0 2.5 1.1t1 2.5v2.1h2.9q1.1 0 2 0.9t0.8 2z"
          />
        </g>
      </svg>
       
      avril 2011 - septembre 2011
    </div>
    <div
      className="c4"
    >
      <p>
        <span
          className="c5"
        >
          Youkan
           :
        </span>
         
        Développement de la troisième version d’une application web de gestion de projet agile en ligne. 
      </p>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.oracle.com/fr/java/"
          target="_blank"
        >
          Java
           6
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://flex.apache.org/"
          target="_blank"
        >
          Flex
           4.5
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://spring.io/"
          target="_blank"
        >
          Spring
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://ibatis.apache.org/"
          target="_blank"
        >
          iBatis
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="http://tomcat.apache.org/"
          target="_blank"
        >
          Tomcat
          
        </a>
      </span>
      <span
        className="c6"
      >
        <a
          className="c7"
          href="https://www.postgresql.org/"
          target="_blank"
        >
          PostgreSQL
          
        </a>
      </span>
    </div>
  </div>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      Master Génie Logiciel 
       (
      Université de Bordeaux
      )
    </div>
    <div
      className="c3"
    >
      <svg
        fill="currentColor"
        height="1em"
        preserveAspectRatio="xMidYMid meet"
        style={
          Object {
            "color": undefined,
            "verticalAlign": "middle",
          }
        }
        viewBox="0 0 40 40"
        width="1em"
      >
        <g>
          <path
            d="m4.4 37.1h6.4v-6.4h-6.4v6.4z m7.8 0h7.2v-6.4h-7.2v6.4z m-7.8-7.8h6.4v-7.2h-6.4v7.2z m7.8 0h7.2v-7.2h-7.2v7.2z m-7.8-8.6h6.4v-6.4h-6.4v6.4z m16.4 16.4h7.1v-6.4h-7.1v6.4z m-8.6-16.4h7.2v-6.4h-7.2v6.4z m17.2 16.4h6.4v-6.4h-6.4v6.4z m-8.6-7.8h7.1v-7.2h-7.1v7.2z m-7.9-19.3v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.4q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.4q0.3 0 0.5-0.2t0.2-0.5z m16.5 19.3h6.4v-7.2h-6.4v7.2z m-8.6-8.6h7.1v-6.4h-7.1v6.4z m8.6 0h6.4v-6.4h-6.4v6.4z m0.7-10.7v-6.4q0-0.3-0.2-0.5t-0.5-0.2h-1.5q-0.3 0-0.5 0.2t-0.2 0.5v6.4q0 0.3 0.2 0.5t0.5 0.2h1.5q0.2 0 0.5-0.2t0.2-0.5z m8.5-1.4v28.5q0 1.2-0.8 2.1t-2 0.8h-31.4q-1.2 0-2.1-0.9t-0.8-2v-28.5q0-1.2 0.8-2t2.1-0.9h2.8v-2.1q0-1.5 1.1-2.6t2.5-1h1.4q1.5 0 2.5 1.1t1.1 2.5v2.1h8.6v-2.1q0-1.5 1-2.6t2.5-1h1.5q1.4 0 2.5 1.1t1 2.5v2.1h2.9q1.1 0 2 0.9t0.8 2z"
          />
        </g>
      </svg>
       
      septembre 2006 - octobre 2011
    </div>
    <div
      className="c4"
    >
      <p>
         
        Préparation d'une Licence puis d'un Master spécialité génie logiciel à l'université de Bordeaux (M. Bien).
      </p>
    </div>
  </div>
</div>
`;
